src/metabase/models/card.clj

```diff
@@ -1053,13 +1053,13 @@
                   :mr         [:model/ModerationReview [:and
                                                         [:= :mr.moderated_item_type "card"]
                                                         [:= :mr.moderated_item_id :this.id]
-                                                        [:= :mr.most_recent true]]]
+                                                        [:= :mr.most_recent true]]]}})
                   ;; Workaround for dataflow :((((((
                   ;; NOTE: disabled for now, as this is not a very important ranker and can afford to have stale data,
                   ;;       and could cause a large increase in the query count for dashboard updates.
                   ;;       (see the test failures when this hook is added back)
                   ;:dashcard  [:model/DashboardCard [:= :dashcard.card_id :this.id]]
-                  }})
+
```

Comment:
This commented out code belongs inside the map. Did you move this intentionally or was it VIM?

Reply:
vim :sad:
do that everytime I open the file. will revert